types.js

```diff
@@ -0,0 +1,2 @@
+"use strict";
```

Comment:
Added the bare minimum required to get EPM to be able to `import '@elastic/package-registry'.`

This is the result when did things [The Right Way](https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html) locally and avoids bringing in TypeScript, running `tsc --declaration ...`, etc each time.

We can move to that later. For now, new golang structs will still be written to `types.d.ts` and available to consumers just by running `mage build`